client/src/leap/soledad/client/encdecpool.py

```diff
@@ -575,6 +578,7 @@ def _get_docs(self, encrypted=None, order_by='idx', order='ASC'):
         query += " ORDER BY %s %s" % (order_by, order)
         return self._runQuery(query)
 
+
```

Comment:
The PEP8 nazism tell me that this extra line is not needed ;P (I don't mind to keep it if you find the code nicer like that)